src/central-server/admin-service/globalconf-generator/src/main/java/org/niis/xroad/cs/admin/globalconf/generator/SharedParameters.java

```diff
@@ -68,6 +67,13 @@ public static class CaInfo {
         private byte[] cert;
     }
 
+    @Data
+    @AllArgsConstructor
+    public static class AcmeServer {
+        private String acmeServerDirectoryURL;
```

Comment:
```suggestion
        private String directoryURL;
```

Reply:
Enclosing class already defines the "acme server" context, no need for `acmeServer.acmeServerDirectory` redundancy.